README.md

```diff
@@ -46,7 +46,6 @@ package main
 import (
```

Comment:
@TheUnderScorer can you also update `Contributing.md` to reflect the changes (examples, description of how the code-gen works, etc) ?

Reply:
Done!